assets/js/googlesitekit/notifications/register-defaults.js

```diff
@@ -479,6 +483,18 @@ export const DEFAULT_NOTIFICATIONS = {
 		},
 		isDismissible: true,
 	},
+	'setup-success-notification-fpm': {
+		Component: FirstPartyModeSetupSuccessSubtleNotification,
+		priority: 310,
```

Comment:
Should we consider prioritising this notification as it is "on-demand"/appears upon action? The IB mentions setting it to a 10, but I'm interested to hear your thoughts, thank you!

Reply:
Hmm yes, I thought I'd added this as a 10, maybe an overenthusiastic AI changed it for me :thinking: 

Anyhow good shout, I've updated this to follow the IB, cheers.